### DISCUSSION TITLE
Can I write a custom policy/package to read terraform files?

### DISCUSSION THREAD

**Discussion by l3112** [aquasecurity/trivy#4211] [Q&A] [answered]
I'm checking out[ this documentation](https://aquasecurity.github.io/trivy/v0.23.0/misconfiguration/custom/) to make a custom policy to read terraform files.

It seems I may have to make a custom package to import to the rego file -- I'm going to try naming a new (rego?) file the package name and putting in the rego code for the policy to call.

However! Then I see this;

`Custom policies do not support Terraform at the moment.`

Is this saying that I **cannot write a package in rego to import to a policy** that checks Terraform files because it does not work, or that I can't write the package in Terraform (Which makes less sense in context but) ?

**Comment by itaysk** [comment]
@simar7 can you ptal?

**Comment by simar7** [comment, marked as answer]
You seem to be using an old version of the docs. The latest version of Trivy supports that and is documented here: https://aquasecurity.github.io/trivy/v0.41/docs/scanner/misconfiguration/custom/

**Reply by l3112** [reply]
Whoops! Thank you!
